apps/web/pages/event-types/[type].tsx

```diff
@@ -1334,6 +1351,120 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
+                                />
+                              )}
+                            />
+
+                            {enableSeats && (
+                              <div className="block sm:flex">
+                                <div className="ml-48 mt-2 inline-flex w-full space-x-2">
+                                  <div className="w-full">
+                                    <Controller
+                                      name="seatsPerTimeSlot"
+                                      control={formMethods.control}
+                                      render={() => {
+                                        const selectSeatsPerTimeSlotOptions = [
+                                          { value: 2, label: "2" },
+                                          { value: 3, label: "3" },
+                                          { value: 4, label: "4" },
+                                          { value: 5, label: "5" },
+                                          {
+                                            value: -1,
+                                            isDisabled: false,
+                                            label: (
+                                              <div className="flex flex-row justify-between">
+                                                <span>6 +</span>
+                                                <Badge variant="default">PRO</Badge>
+                                              </div>
+                                            ),
+                                          },
+                                        ];
+                                        return (
+                                          <>
+                                            <div className="block sm:flex">
+                                              <div className="flex-auto">
+                                                <label
+                                                  htmlFor="beforeBufferTime"
+                                                  className="mb-2 flex text-sm font-medium text-neutral-700">
+                                                  Number of seats per booking
+                                                </label>
+                                                <Select
+                                                  isSearchable={false}
+                                                  classNamePrefix="react-select"
+                                                  className="react-select-container focus:border-primary-500 focus:ring-primary-500 block w-full min-w-0 flex-auto rounded-sm border border-gray-300 sm:text-sm "
+                                                  onChange={(val) => {
+                                                    if (val!.value === -1) {
+                                                      formMethods.setValue("seatsPerTimeSlot", 6);
+                                                      setInputSeatNumber(true);
```

Comment:
If someone choose the "6+" seat option it should then set the minimum seats as 6